assets/migrations/run_mybatis_migrations.sh

```diff
@@ -1,13 +1,8 @@
 #!/bin/bash
 curl -L  "https://github.com/mybatis/migrations/releases/download/mybatis-migrations-3.3.1/mybatis-migrations-3.3.1-bundle.zip" > /tmp/mybatis-migrations-3.3.1-bundle.zip
-mkdir -p /u01/postgres/core
-mkdir  /u01/postgres/error
-mkdir  /u01/postgres/schedule
-mkdir  /u01/postgres/feed
-mkdir  /u01/postgres/form
-chown -R postgres /u01/postgres
+
 unzip /tmp/mybatis-migrations-3.3.1-bundle.zip -d /opt/mybatis
 export MIGRATIONS_HOME=/opt/mybatis/mybatis-migrations-3.3.1
 MIGRATIONS=$MIGRATIONS_HOME/bin
 export PATH=$MIGRATIONS:$PATH
-migrate  up --path=configs/assets/migrations  --env=test --force 
+migrate  up --path=configs/assets/migrations  --env=test
```

Comment:
@githengi how do you think we should test if the migrations are idempotent? Maybe run the `migrate up` command twice?

Reply:
Okay, yes sure will add that